### PR TITLE
Profile routing

### PR DESCRIPTION
**Pull request by Mellifaro**
Created models, messages, router and handlers for profile type of requsts. I haven't implemented method handle() in handlers yet.

### CODE REVIEW

File: services/profile/profile-api/src/main/java/com/softgroup/profile/api/message/getContactProfiles/ContactProfilesResponse.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import com.softgroup.common.protocol.ResponseData;
+import com.softgroup.profile.api.model.Profile;
+
+import java.util.List;
+
+/**
+ * Created by Виктор on 13.03.2017.
+ */
+public class ContactProfilesResponse implements ResponseData {
```
Comment (Dominys) [inline services/profile/profile-api/src/main/java/com/softgroup/profile/api/message/getContactProfiles/ContactProfilesResponse.java]: @Mellifaro Read API. It's called Get!

File: services/profile/profile-api/src/main/java/com/softgroup/profile/api/model/Contact.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.softgroup.profile.api.model;
+
+/**
+ * Created by Виктор on 13.03.2017.
+ */
+public class Contact {
```
Comment (Dominys) [inline services/profile/profile-api/src/main/java/com/softgroup/profile/api/model/Contact.java:6]: @Mellifaro All DTO classes must have DTO suffix.

File: services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/ContactsSynchronizeHandler.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import com.softgroup.profile.api.message.contactSync.ContactSynchronizeRequest;
+import com.softgroup.profile.api.message.contactSync.ContactSynchronizeResponse;
+import com.softgroup.profile.api.router.ProfileRequestHandler;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by Виктор on 13.03.2017.
+ */
+@Component
+public class ContactsSynchronizeHandler extends AbstractRequestHandler<ContactSynchronizeRequest, ContactSynchronizeResponse>
```
Comment (Dominys) [inline services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/ContactsSynchronizeHandler.java]: @Mellifaro How this is named in api ?

**Comment by Dominys** [comment]
@Mellifaro Rename Requests/Responses accordingly to their commands.

**Comment by Mellifaro** [comment]
ok. I will correct them.

**Comment by Mellifaro** [comment]
Renamed Requests/Responses accordingly to their commands.

**Review by Dominys — APPROVED** [review]

**Pull request merged**